src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -8,6 +8,10 @@ public class Bulldozer extends Machine {
     public Bulldozer() {
     }
 
+    public Bulldozer(String name, String color, int horsePower) {
+        super();
```

Comment:
Values of fields `name`, `color` and `horsePower` are just ignored.
Please, create  constructor with 3 parameters in `Machine` class and set it (ex this. name = name; ...)